### PR TITLE
completed exercise html-forms

### PR DESCRIPTION
**Pull request by marissalortega**
htmlforms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="http://www.example.com/subscribe" method="post">
+      <div>
+        <label for="email">
+          <div>Email</div>
+          <input type="email" name="email" maxlength="100">
```
Comment (BrettAlbright) [inline html-forms/index.html]: this input needs an `id` property to match the `for` property of the label

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      </div>
+      </label>
+      </div>
+
+      <div>
+        <label for="why-coding">
+          <div>Why is coding cool?</div>
+          <select name="coding">
```
Comment (BrettAlbright) [inline html-forms/index.html]: same as above

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        <label for="javascript">
+          <input type="radio" name="language" value="javascript" />
+          JavaScript
+        </label>
+      </div>
+
+      <div>
+        <p>Would you like to sign up for more information?</p>
+        <label for="signup">
+          <input type="checkbox" name="sign-up" />
```
Comment (BrettAlbright) [inline html-forms/index.html]: same as above

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        </label>
+      </div>
+
+      <div>
+        <div>
+          <label for="comments">
+            Comments
+          </label>
+        </div>
+        <textarea name="comments" cols="30" rows="10">
```
Comment (BrettAlbright) [inline html-forms/index.html]: same as above

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Very close! Just a few small updates!

**Comment by marissalortega** [comment]
htmlforms

Still has the same functionality, but I added the missing `id` properties.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**